### PR TITLE
Restructure the main page

### PR DESCRIPTION
**Pull request by mrkara**
To make room for the new user guides.

* Add 2 new categories for the users and the node operators.
* Try to arrange from top to bottom for users > mainnet > testnet
* Add guides from notion:
  * Delegating CSPR Stake
  * Signer Guide
  * Generate Keys
  * Connect a Wallet

### CODE REVIEW

**Comment by mrkara** [comment]
Let's not merge this yet.

File: README.md [hunk 1]
```diff
@@ -1,26 +1,32 @@
...
-This documentation is aimed to help prospective node-operators set up and configure their Casper Network validator node.
+## For Users
 
-## Testnet Information
-  - [Test Net Code of Conduct and Incentive Requirements](/docs/testnet.md)
-  - [Testnet Rewards](/docs/testnet-rewards.md)
-  - [IMPORTANT - Testnet Upgrade to casper-node v1.1.0 before April 22](/docs/testnet/upgrade-1_1_0.md)
-  - [IMPORTANT - Testnet Upgrade to casper-node v1.1.2 before May 11](/docs/testnet/upgrade-1_1_2.md)
-  - [IMPORTANT - Testnet Upgrade to casper-node v1.2.0 before May 19](/docs/testnet/upgrade-1_2_0.md)
+This documentation aims to help with onboarding the new citizens of the Casper network by providing guides on basic setup and operations.
```
Comment (mssteuer) [inline README.md:5]: 1. let's replace "citizens" with "end users" 
2. capitalize "Casper Network"

**Review by mssteuer — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mrkara) [reply README.md]: :+1: Done.

File: docs/user-guides/Connect-a-Wallet.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# Connect a Wallet
+
+## What You'll Need
+
+A compatible CSPR wallet: either [CasperLabs's Signer](https://www.notion.so/Signer-Guide-eb8c504d067344109351bc0f21ac2412) or [Ledger](https://www.ledger.com/). More wallets are planned for the future.
```
Comment (mssteuer) [inline docs/user-guides/Connect-a-Wallet.md:5]: I would remove the Ledger for now, as it is not available yet. Maybe "More wallets, including Ledger, are planned for the future" ?

File: docs/user-guides/Connect-a-Wallet.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## Sign In
+
+You can access the wallet Sign In screen by clicking `Sign In` from the top navigation menu. Then choose your Sign In option, either Signer or Ledger:
```
Comment (mssteuer) [inline docs/user-guides/Connect-a-Wallet.md:9]: Revise this to not offer Ledger as an option. It's not available yet. We can build this back in when the Ledger is available.

File: docs/user-guides/Delegating-CSPR-Stake.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+# Delegating CSPR Stake
+
+## What You'll Need
+
+A compatible wallet with a CSPR balance: either [CasperLabs's Signer](https://chrome.google.com/webstore/detail/casperlabs-signer/djhndpllfiibmcdbnmaaahkhchcoijce) or [Ledger](https://www.ledger.com/). More wallets are planned for the future.
```
Comment (mssteuer) [inline docs/user-guides/Delegating-CSPR-Stake.md:5]: No Ledger currently

File: docs/user-guides/Delegating-CSPR-Stake.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+## The Basics
+
+### Connect a Wallet
+
+You can access the wallet Sign In screen by clicking `Sign In` from the top navigation menu. Then choose your Sign In option, either Signer or Ledger:
```
Comment (mssteuer) [inline docs/user-guides/Delegating-CSPR-Stake.md:11]: No Ledger currently

File: docs/user-guides/Delegating-CSPR-Stake.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+#### ...Using Signer
+
+1. From the Sign In screen click `Sign In` under the Casper Signer option
+2. The Signer app window will open automatically. From the Signer window, when prompted to connect Signer to site, click `Connect` . Then click `Connect` again to approve the connection.
+3. Select an account to connect to.
+
+#### ...Using Ledger
```
Comment (mssteuer) [inline docs/user-guides/Delegating-CSPR-Stake.md:19]: No Ledger currently. Do we want to comment all this out so it's available for re-inclusion later?

File: docs/user-guides/SignerGuide.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
+# Signer Guide
+
+![Casper Signer Logo](../../assets/CasperNetwork/casper-signer-logo.jpg)
+
+## New User (No Accounts)
+
+### Download Signer
+
+Using Chrome or a Chromium-based browser like Brave, visit the Chrome Web Store, then download and install the [Casper Labs Signer extension](https://chrome.google.com/webstore/detail/casperlabs-signer/djhndpllfiibmcdbnmaaahkhchcoijce).
```
Comment (mssteuer) [inline docs/user-guides/SignerGuide.md:9]: "CasperLabs", not "Casper Labs"

File: docs/user-guides/SignerGuide.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+### Set Up Signer
+
+Signer let's you sign deploys on the Casper network. To get started, you'll need to create a Vault.
```
Comment (mssteuer) [inline docs/user-guides/SignerGuide.md:13]: "Casper Network"

**Review by mssteuer — CHANGES_REQUESTED** [review]

File: docs/user-guides/Connect-a-Wallet.md (hunk 2 above)
Comment (mrkara) [reply docs/user-guides/Connect-a-Wallet.md:5]: Done.

File: docs/user-guides/Connect-a-Wallet.md (hunk 3 above)
Comment (mrkara) [reply docs/user-guides/Connect-a-Wallet.md:9]: :+1: Done.

File: docs/user-guides/Delegating-CSPR-Stake.md (hunk 6 above)
Comment (mrkara) [reply docs/user-guides/Delegating-CSPR-Stake.md:19]: :+1: We will have it in the commit and commenting out in markdown seems a bit hack/messy. I think we can just remove it for now.

File: docs/user-guides/Delegating-CSPR-Stake.md (hunk 4 above)
Comment (mrkara) [reply docs/user-guides/Delegating-CSPR-Stake.md:5]: :+1:

File: docs/user-guides/Delegating-CSPR-Stake.md (hunk 5 above)
Comment (mrkara) [reply docs/user-guides/Delegating-CSPR-Stake.md:11]: :+1:

File: docs/user-guides/SignerGuide.md (hunk 7 above)
Comment (mrkara) [reply docs/user-guides/SignerGuide.md:9]: :+1:

File: docs/user-guides/SignerGuide.md (hunk 8 above)
Comment (mrkara) [reply docs/user-guides/SignerGuide.md:13]: :+1:

**Review by mssteuer — APPROVED** [review]

**Pull request merged**